core/trino-main/src/main/java/io/trino/type/DecimalCasts.java

```diff
@@ -457,15 +482,27 @@ public static Slice realToLongDecimal(long value, long precision, long scale, Bi
     }
 
     @UsedByGeneratedCode
-    public static Slice shortDecimalToVarchar(long decimal, long precision, long scale, long tenToScale)
+    public static Slice shortDecimalToVarchar(long decimal, long scale, long length)
```

Comment:
length -> varcharLength

(otherwise, by looking at method signature, it's unclear what "length" it's referring to)